sklearn/metrics/classification.py

```diff
@@ -1994,3 +1994,123 @@ def brier_score_loss(y_true, y_prob, sample_weight=None, pos_label=None):
     y_true = np.array(y_true == pos_label, int)
     y_true = _check_binary_probabilistic_predictions(y_true, y_prob)
     return np.average((y_true - y_prob) ** 2, weights=sample_weight)
+
+
+def calibration_loss(y_true, y_prob, reducer="sum", nbins=10, normalize=True,
+                     sample_weight=None, pos_label=None):
+    """Compute  calibration loss.
+
+    Across all items in a set N predictions, the calibration loss measures
```

Comment:
We usually put most of this detail in the user guide, not in API reference

Reply:
Thanks for the suggestion, I'll move it there then.